libs/coin-modules/coin-bitcoin/src/wallet-btc/__tests__/xpub.txs.P2WPKH.integration.test.ts

```diff
@@ -105,6 +105,8 @@ describe.skip("testing xpub native segwit transactions", () => {
         .toString("hex");
 
       psbt.addInput({
+        // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+        //@ts-expect-error
```

Comment:
better to add a comment here after the expect-error, we can use this rule 

![CleanShot 2024-04-03 at 15 16 08@2x](https://github.com/LedgerHQ/ledger-live/assets/671786/d57859b5-3df6-40ea-bdcc-c2c333a29f66)

Reply:
Thanks, I will update it